libs/redux/clientApi.ts

```diff
@@ -32,7 +32,7 @@ const clientApi = createApi({
    * Tag types must be defined in the original API definition
    * for any tags that would be provided by injected endpoints
    */
-  tagTypes: ['TerrainsApi', 'UserApi'],
+  tagTypes: ['TerrainsApi', 'UserTerriansAPI','UserGuardiansAPI', 'UserApi'],
```

Comment:
This is altering 2 `tagTypes` instead of just 1.

The naming convention should be a capitalized camelCase action name with `Api` instead of 'API`. 
Ex: `UserTerrainsApi`, `UserGuardiansApi`